src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/multinomial.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/runtime/tensor.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+
+using namespace ov::test;
```

Comment:
Please avoid using `using namespace` in headers

Reply:
Fixed!